gamestonk_terminal/cryptocurrency/overview/coinpaprika_model.py

```diff
@@ -0,0 +1,295 @@
+from datetime import datetime
+import textwrap
+import pandas as pd
+from dateutil import parser
+from gamestonk_terminal.cryptocurrency.coinpaprika_helpers import (
+    ENDPOINTS,
+    PaprikaSession,
+)
+
+
+session = PaprikaSession()
+
+
+def get_global_market():
+    """Return data frame with most important global crypto statistics like:
+    market_cap_usd, volume_24h_usd, bitcoin_dominance_percentage, cryptocurrencies_number,
+    market_cap_ath_value, market_cap_ath_date, volume_24h_ath_value, volume_24h_ath_date,
+    market_cap_change_24h, volume_24h_change_24h, last_updated,
+
+    Returns
+    -------
+    pandas.DataFrame
+        Metric, Value
+    """
+    global_markets = session.make_request(ENDPOINTS["global"])
+    global_markets["last_updated"] = datetime.fromtimestamp(
+        global_markets["last_updated"]
+    )
+
+    for key, date in global_markets.items():
+        if "date" in key:
+            try:
+                global_markets[key] = parser.parse(date).strftime("%Y-%m-%d %H:%M:%S")
+            except (KeyError, ValueError, TypeError):
+                ...
```

Comment:
I have not seen this before `...`. What does it mean?

Reply:
It's probably my bad habbit, It's very similar to just `pass` statement. Probably I missed that one, I will add there print statement to know what failed.

Reply:
I had not seen `...` before! Interesting. But yes, ideally we would output what is the error. This way in case it crashes the user can tell us what is the message they saw, and we can know what happened.

You can do something like
```
except (KeyError, ValueError, TypeError) as e:
    print(e)
```

So that you still capture the error without the terminal crashing!